### PR TITLE
PE-3773: upgrade flutter to v3.10.0 - relates to PE-3787

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
Upgraded Flutter and Storybook

### CODE REVIEW

**Review by karlprieb — CHANGES_REQUESTED** [review]
Update fvm settings

**Comment by thiagocarvalhodev** [comment]
> Update fvm settings

https://github.com/ar-io/ardrive_ui/pull/36/commits/584de047b61e183e25be5a37741bb2ff78c73aef

**Review by karlprieb — DISMISSED** [review]

File: lib/src/styles/theme/themes.dart [hunk 1]
```diff
@@ -44,6 +44,7 @@ class _ArDriveAppState extends State<ArDriveApp> {
   void initState() {
     super.initState();
 
+    // ignore: deprecated_member_use
     var window = WidgetsBinding.instance.window;
```
Comment (matibat) [inline lib/src/styles/theme/themes.dart:48]: I know there are more ways of accessing this object, we'll have to have a tech_debt

https://docs.flutter.dev/release/breaking-changes/window-singleton

File: pubspec.yaml [hunk 2]
```diff
@@ -7,13 +7,13 @@ publish_to: "none"
...
   sdk: ">=2.18.5 <3.0.0"
-  flutter: 3.3.9
+  flutter: 3.10.0
 
 dependencies:
   ardrive_io:
     git:
       url: https://github.com/ar-io/ardrive_io.git
-      ref: v1.2.0
+      ref: PE-3773-upgrade-flutter-to-v-3-10-0
```
Comment (matibat) [inline pubspec.yaml:16]: Reminder to update this when released

File: storybook/pubspec.yaml [hunk 3]
```diff
@@ -7,15 +7,15 @@ version: 1.0.0+1
...
+  flutter: ">=3.10.0"
 
 dependencies:
   flutter:
     sdk: flutter
   cupertino_icons: ^1.0.2
   ardrive_ui:
     path: ../
-  widgetbook: ^2.4.1
+  widgetbook: 3.0.0-beta.14
```
Comment (matibat) [inline storybook/pubspec.yaml:18]: A beta? Was this intended?

**Review by matibat — APPROVED** [review]

**Review by karlprieb — DISMISSED** [review]

File: pubspec.yaml [hunk 4]
```diff
@@ -7,13 +7,13 @@ publish_to: "none"
...
   sdk: ">=2.18.5 <3.0.0"
-  flutter: 3.3.9
+  flutter: 3.10.0
 
 dependencies:
   ardrive_io:
     git:
       url: https://github.com/ar-io/ardrive_io.git
-      ref: v1.2.0
+      ref: PE-3934-modal-does-not-close-after-successful-or-failed-file-download-on-mobile
```
Comment (matibat) [inline pubspec.yaml:16]: Don't we have to update this? - I'm not sure if it's already released

**Review by matibat — APPROVED** [review]

File: pubspec.yaml (hunk 4 above)
Comment (thiagocarvalhodev) [reply pubspec.yaml:16]: > Don't we have to update this? - I'm not sure if it's already released
you right! I'll update the PR

**Review by matibat — APPROVED** [review]
LGTM thanks!

**Pull request merged**